data.sql

```diff
@@ -4,3 +4,19 @@ VALUES
   ('Gabumon', '2018-11-15', 2, TRUE, 8),
   ('Pikachu', '2021-01-07', 1, FALSE, 15.04),
   ('Devimon', '2017-05-12', 5, TRUE, 11);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES
+  ('Charmander', '2020-02-08', 0, FALSE, -11),
+  ('Plantmon', '2021-11-15', 2, TRUE, -5.7),
+  ('Squirtle', '1993-04-02', 3, FALSE, -12.13),
+  ('Angemon', '2005-06-12', 1, TRUE, -45),
+  ('Boarmon', '2005-06-07', 7, TRUE, 20.4),
+  ('Blossom', '1998-10-13', 3, TRUE, 17),
+  ('Ditto', '2022-05-14', 4, TRUE, 22);
+
+BEGIN;
+ALTER TABLE animals
+RENAME COLUMN species TO unspecified;
```

Comment:
- [ ] Here we are supposed to set the fields of the `species` column to unspecified not to rename the column itself to unspecified. e.g.

```
UPDATE animals
   SET species = 'unspecified'; 
```
Kindly fix this
![image](https://github.com/torobucci/vet_clinic/assets/81932478/9eb8b1f1-6689-4b1f-974d-8b43975c72fa)